backend/app/usecase/feature/decision.go

```diff
@@ -1,11 +1,27 @@
 package feature
 
-import "github.com/short-d/short/backend/app/usecase/instrumentation"
+import (
+	"github.com/short-d/short/backend/app/entity"
+	"github.com/short-d/short/backend/app/usecase/instrumentation"
+)
+
+const (
+	ChangeLog string = "change-log"
+	FacebookSignIn string = "facebook-sign-in"
+	GithubSignIn string = "github-sign-in"
+	GoogleSignIn string = "google-sign-in"
+	SearchBar string = "search-bar"
+	UserShortLinksSection string = "user-short-links-section"
+	PreferenceToggles string = "preference-toggles"
```

Comment:
I think those are not permission toggles, right? Should we handle them with special care instead of directly fetching DB response?